test/e2e/examples/index.test.ts

```diff
@@ -0,0 +1,47 @@
+import { createNextDescribe } from 'e2e-utils'
+import path from 'path'
+import fs from 'fs-extra'
+
+const testedExamples = [
+  'active-class-name',
+  'amp',
+  'amp-first',
+  'amp-story',
+  'api-routes',
+  'api-routes-cors',
+  'api-routes-middleware',
+]
+
+testedExamples.forEach((example) => {
+  const exampleFiles = path.join(
+    __dirname,
+    '..',
+    '..',
+    '..',
+    'examples',
+    example
+  )
+
+  const packageJson = fs.readJsonSync(path.join(exampleFiles, 'package.json'))
+
+  createNextDescribe(
+    `example '${example}'`,
+    {
+      files: exampleFiles,
+      dependencies: {
```

Comment:
`devDependencies` should be taken into consideration here as well.

Reply:
:+1: